torch/_inductor/ir.py

```diff
@@ -4367,7 +4367,7 @@ def should_allocate(self):
 
     # TODO: handle bools carefully
     def __init__(self, sym, data):
-        super().__init__(None, NoneLayout(torch.device("cpu")), [data])  # type: ignore[arg-type]
+        super().__init__(None, NoneLayout(torch.device("cpu")), self.unwrap_storage([data]))  # type: ignore[arg-type]
```

Comment:
Is the ignore[arg-type] still needed after the unwrap?

Reply:
Yeah, I believe the arg-type is for NoneLayout which does not inherit Layout